src/api/spec/lib/backend/api/cloud_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Backend::Api::Cloud do
+  context '#upload' do
+    let(:user) { create(:confirmed_user) }
+    let!(:ec2_configuration) { create(:ec2_configuration, user: user) }
+
+    context 'with a single string' do
```

Comment:
with a single string?

Reply:
Not sure how this got there. Probably a c&p mistake